plugins/out_opentelemetry/opentelemetry.c

```diff
@@ -1348,6 +1494,57 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct opentelemetry_context, log_response_payload),
      "Specify if the response paylod should be logged or not"
     },
+    {
+     FLB_CONFIG_MAP_STR, "logs_observed_timestamp_metadata_key", "$ObservedTimestamp",
+     0, FLB_TRUE, offsetof(struct opentelemetry_context, logs_observed_timestamp_metadata_key),
+     "Specify an ObservedTimestamp key"
```

Comment:
Since these are using record accessors now, is `_key` still the right word to use for the config field name?

Reply:
I think it is right word.

Some plugins support a record accessor as a `key`.
e.g. https://docs.fluentbit.io/manual/pipeline/filters/rewrite-tag#key

It allows that user doesn't need to take care that a key is nested or not.